WIP PR to help collect feedback on draft (#3)
ryancdickson/www.ccadb.org: policy.md

CA Owners MUST disclose:
- all subordinate CA certificates capable of validating to a certificate included in a Root Store or associated with a CCADB Root Inclusion Request. Disclosure MUST take place within 7 calendar days of issuance and before the subject CA represented in the certificate begins issuing publicly-trusted certificates.

> - subordinate CA ownership using the "Subordinate CA Owner" field, as described in its tooltip.

Reviewer: ```suggestion
- subordinate CA ownership using the "Subordinate CA Owner" field. This is the subordinate CA Owner's name as it appears in the provided audit statements. CA Owners MUST leave this field blank if and only if both control of the private key and domain/IP control validation activities are performed by the organization listed in the audit statement of the parent certificate.
```
The tooltip is not visible to everyone. I think the risk of potential skew between the policy and the tooltip is relatively low and this is a fairly important accountability requirement. I revert my previous suggestion.

> - subordinate CA ownership using the "Subordinate CA Owner" field. This is the subordinate CA Owner's name as it appears in the provided audit statements. CA Owners MUST leave this field blank if and only if both control of the private key and domain/IP/email validation activities are performed by the organization listed in the audit statement of the parent certificate.

Reviewer: meta question: If something like this is already conveyed within the CCADB either through page layout header guidance or a tooltip or similar - do we want to repeat that language in the CCADB Policy? To me, the CCADB Policy is best for describing things that must go into the CCADB that are not already easily explained in the page layout, like the line above needing to disclose new ICAs and along what timeline. Having the language exist in two places creates opportunities for skew.

Author: So, let's drop:

"This is the subordinate CA Owner's name as it appears in the provided audit statements. CA Owners MUST leave this field blank if and only if both control of the private key and domain/IP/email validation activities are performed by the organization listed in the audit statement of the parent certificate."

?

Reviewer: Maybe we should update the tooltip with Andrew's suggest edit and then change this to:

```suggestion
- subordinate CA ownership using the "Subordinate CA Owner" field, as described in it's tooltip.
```

I fear just leaving "subordinate CA ownership using the "Subordinate CA Owner" field." would result in CA Owners adding their own name incorrectly, as we've seen recently through several CRS inclusion detailed reviews.

    - freely publicly available for examination.
    - available in an authoritative English language version.
    - sufficiently detailed to assess the operations of the CA(s) and the compliance with the expectations set forth in this Policy, the applicable CA/Browser Forum Baseline Requirements, and any applicable Root Store Operator policies, and MUST NOT conflict with any of the requirements specified therein.
- effective **TBD**, identify the in-scope CAs by including a table or bulleted list in Section 1.3.1 that minimally includes:
    - The in-scope certificate's Subject Name,
    - The in-scope certificate's SHA-256 fingerprint, and
    - The set of PKI use cases served.
- effective **TBD**, state adherance to this policy, and any applicable ecosystem policy the corresponding PKI hierarchy(ies) intends to comply with.

> - effective **TBD**, explicitly and clearly describe the relationship between the document and any other applicable policy documents within the hierarchy (e.g., a superior CA's CP/CPS), including the order of precedence in the event of a conflict.

Reviewer: ```suggestion

Effective **TBD**, CA policy documents MUST:
- identify the in-scope CAs by including a table or bulleted list in Section 1.3.1 that minimally includes the:
    - in-scope certificate's Subject Name,
    - in-scope certificate's SHA-256 fingerprint, and
    - set of PKI use cases served.
- state adherence to this policy, and any applicable ecosystem policy the corresponding PKI hierarchy(ies) intends to comply with.
- explicitly and clearly describe the relationship between the document and any other applicable policy documents within the hierarchy (e.g., a superior CA's CP/CPS), including the order of precedence in the event of a conflict.
```
Once this date passes, these bullets can roll up into the list above.

    - available in an authoritative English language version.
    - sufficiently detailed to assess the operations of the CA(s) and the compliance with the expectations set forth in this Policy, the applicable CA/Browser Forum Baseline Requirements, and any applicable Root Store Operator policies, and MUST NOT conflict with any of the requirements specified therein.

Effective September 15, 2026, CA policy documents MUST:
- identify the in-scope CAs by including a table or bulleted list in Section 1.3.1 that minimally includes the:
    - in-scope certificate's Subject Name,
    - in-scope certificate's SHA-256 fingerprint, and
    - set of PKI use cases served.
- state adherence to this policy, and any applicable ecosystem policy the corresponding PKI hierarchy(ies) intends to comply with.
- explicitly and clearly describe the relationship between the document and any other applicable policy documents within the hierarchy (e.g., a superior CA's CP/CPS), including the order of precedence in the event of a conflict.

> To promote simplicity and clarity, all CA policy documents SHOULD:

Reviewer 2: If the documents SHOULD be focused on one specific PKI use case, the exact use cases will need to be listed to avoid confusion, including EKUs that are in scope for each use case. In the case of Apple, we use the term "purpose" in relation to use cases, but those purposes could have different combinations of EKUs or trust bits:
TLS (serverAuth OR serverAuth and clientAuth)
S/MIME (emailProtection OR emailProtection and clientAuth)
Client Auth (clientAuth, but not serverAuth, emailProtection, or timeStamping)
Timestamping (timeStamping and not contain the same public key as any other certificate which asserts any other EKU OIDs)

Since other Root Programs may have different names for these use cases, with different EKUs, it seems like the right time to align on standardized terms that we all use that support all of our different goals. Perhaps expanding into more specific use case labels such as:
TLS Server Authentication-only (serverAuth only)
TLS Server Authentication and Client Authentication (serverAuth and clientAuth only)
S/MIME-only (emailProtection only)
S/MIME and Client Authentication (emailProtection and clientAuth only)
Code Signing-only (codeSigning only)
Code Signing and Time Stamping (codeSigning and timeStamping only)
Client Authentication-only (clientAuth only)
BIMI-only (BrandIndicatorforMessageIdentification only)

Should there be different labels when clientAuth is used along with another EKU? 

What other combinations are there that we recommend get split into separate focused CP/CPS documents?

11. Date the audit statement was issued (referred to as the "issuing date" by the ACAB'c Audit Attestation Letter templates and "report date" by the WebTrust Illustrative Reports);
    - For revised or corrected versions of audit statements made to address errors, omissions, or formatting to a previously completed audit statement, the CA Owner may enter the date of the initial issuance, provided that the scope and audit period covered by the revised audit statement remain unchanged. In such cases, the audit statement should contain the initial issuance date and the subsequent issuance date and the CA Owner should provide the URL of the most recent version and include a comment in the case explaining the revision date and the reason for reissuance of the audit statement.
12. For ETSI, a statement to indicate if the audit was a full audit, and which parts of the criteria were applied, e.g. DVCP, OVCP, NCP, NCP+, LCP, EVCP, EVCP+, QCP-w, Part1 (General Requirements), and/or Part 2 (Requirements for trust service providers), and a statement to indicate that the auditor referenced the applicable CA/Browser Forum criteria and the version used;

> 13. All [incidents](#61-disclosing-and-responding-to-incidents) disclosed by the CA Owner, or reported by a third party, and all findings reported by an auditor, that, at any time during the audit period, occurred, were open in Bugzilla, or were reported to a Root Store Operator. For each, auditors SHOULD opine that (a) the scope, impact, and root cause of incidents are accurately and fairly stated in the publicly-disclosed incident reports, and (b) that the corresponding actions taken by the CA Owner satisfactorily address those root causes and meaningfully reduce likelihood of the issue’s recurrence; and

Reviewer 3: I'd rephrase this to sound more neutral by replacing the word "opine" and also less predictive in terms of remediation ("meaningfully reduce likelihood of recurrence"). Here is some alternative language we could consider.
"For each, auditors SHOULD review the publicly-disclosed incident reports for consistency with audit evidence obtained" ... "and indicate [whether/that] (a) the scope, impact, and root cause are accurately and fairly described;" and "[whether/that] (b) the corrective actions described by the CA Owner are aligned with the factors that led to the incident and are intended to mitigate the risks associated with the identified root cause(s)."

Author: Incorporated in https://github.com/ryancdickson/www.ccadb.org/commit/119f57da1d44a6c59d24aaa7d61791c970fd3154.

- 2.23.140.1.2.3
- 2.23.140.1.1

> For hierarchies used to issue Code Signing and/pr Time Stamping certificates:

Reviewer: ```suggestion
For hierarchies used to issue Code Signing and/or Time Stamping certificates:
```

### 6.2 Certificate Revocation List Disclosures

> For any unexpired and unrevoked CA certificate disclosed to the CCADB, CA Owners MUST disclose the complete set of distinct HTTP URLs appearing in the `crlDistributionPoints` extension of the unexpired certificates issued by that CA. The disclosed URLs MUST match exactly as they appear in the issued certificates.

Reviewer 3: What about saying, ", CA Owners MUST disclose**, in a JSON array,** the complete set ..."?

Author: Is https://github.com/ryancdickson/www.ccadb.org/commit/6763710733aa558444cdd0911318dbc49b5ef1de what you had in mind?

### 6.2 Certificate Revocation List Disclosures

For any unexpired and unrevoked CA certificate disclosed to the CCADB, CA Owners MUST disclose the complete set of distinct HTTP URLs appearing in the `crlDistributionPoints` extension of the unexpired certificates issued by that CA. The disclosed URLs MUST match exactly as they appear in the issued certificates.

CA Owners MUST disclose these URLs to the CCADB at the earlier of:
- within 7 days of the corresponding CA issuing the first certificate containing the URL; OR
- within 4 hours of the corresponding CA revoking the first certificate containing the URL.

If the CA has not yet issued any certificates (i.e., no CRL URLs are established), the CA Owner MUST enter `[""]` into the "JSON Array of Partitioned CRLs" field until issuance begins.

If the CA certificate is expired and there are no longer unexpired leaf certificates validating to it, the CA Owner MUST enter the string `expired` into the "All Full CRL URIs for This Hierarchy" field.

> If the CA certificate is revoked, no `crlDistributionPoints` disclosures are required.

Reviewer: ```suggestion
If the CA certificate is revoked, no `crlDistributionPoints` disclosures are required on that certificate record.
```
